### ISSUE TITLE
Ambiguous Ingredient Quantity

### ISSUE THREAD

**Issue by lytoooo0** [LukeSmithxyz/based.cooking#918]
The recipe of [Korv Stroganoff ](https://based.cooking/korv-stroganoff/) suggests to add "2 1/2 dl cream", which is a little bit confusing, is it 2.5, 2 or 1/2? @axelf4

**Comment by axelf4** [comment]
It's 2.5 dl. The use of mixed numbers in recipes is fairly established, I think.

**Comment by cassiofb-dev** [comment]
Yeah, mixed number are often used in my country (including recipes too). The mixed number "2 1/2" is pronounced as "2 and a half", the course below is a good way to get started with mixed numbers (I love Khan Academy, finished a lot of courses):

https://www.khanacademy.org/math/cc-fourth-grade-math/imp-fractions-2/imp-mixed-numbers/a/mixed-numbers-and-improper-fractions-review
